public/app/percona/dbaas/components/DBCluster/ResourcesBar/ResourcesBar.tsx

```diff
@@ -45,32 +45,29 @@ export const ResourcesBar: FC<ResourcesBarProps> = ({
             {Messages.buildResourcesLabel(formatResources(allocated), allocatedWidth, formatResources(total), units)}
           </span>
         )}
-        {isResourceInsufficient ? (
+        {allocated && !isResourceInsufficient && (
```

Comment:
Yes that't exactly it! When the user hasn't selected a kubernetes cluster yet, none of the conditions will pass, hence it will show just an empty component (with only the icon and the bar). Also the div adds equal margin and height, it would be trickier to do without it, would have to add margin to the content.

Reply:
Got it 😄